app.py

```diff
@@ -1,37 +1,118 @@
 from flask import Flask, render_template, request, redirect, url_for
-from item import TrelloItem
-from trello import TrelloClient
+from flask_login import LoginManager, login_required, login_user, current_user
+from oauthlib.oauth2 import WebApplicationClient
 from viewmodel import ViewModel
+from mongo_query import get_items, mark_item_as_complete, add_new_item, mark_item_as_uncomplete
+from user import User
+from userRole import UserRole
 import os
+import pymongo
+import requests
 
+def get_db_collection():
+
+    dbClientUri = os.getenv('MONGO_DB_CONNECTION')
+    databaseName = os.getenv('MONGO_DB_DATABASE')
+    collectionName = 'collection'
+
+    dbClient = pymongo.MongoClient(dbClientUri)
+    db = dbClient[databaseName]
+    return db[collectionName]
 
 def create_app():
     app = Flask(__name__)
 
-    client = TrelloClient(
-            os.getenv('API_KEY'),
-            os.getenv('TOKEN')
+    collection = get_db_collection()
+    
+    os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
```

Comment:
Rather than hardcoding this, we should add it into our local environment variables (e.g. .env file, or via Heroku) - if one environment is happy to ban insecure protocols then we shouldn't be circumventing that